src/main/java/seedu/address/model/appointment/AppointmentContainsKeywordsPredicate.java

```diff
@@ -54,8 +54,7 @@ public boolean test(Appointment appointment) {
                 .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(
                         model.getPersonById(
                                 appointment.getPersonId()).getName().fullName,
-                        keyword)
-                        || appointment.getPersonIdString().equals(keyword));
```

Comment:
Necessary for clicking on person card to find their appointments. Please revert.